releasenotes/notes/fix-initial-point-with-reps-6157ede7a3b9924a.yaml

```diff
@@ -0,0 +1,9 @@
+---
+deprecations:
+  - |
+    The public :attr:`excitation_list` property has now been removed from
+    :class:`qiskit_nature.second_q.algorithms.initial_points.HFInitialPoint`,
+    :class:`qiskit_nature.second_q.algorithms.initial_points.MP2InitialPoint`, and
+    :class:`qiskit_nature.second_q.algorithms.initial_points.VSCFInitialPoint`.
+    Thus the excitation list cannot be set directly, and the private attribute
+    :attr:`_excitation_list` is now set exclusively via the :attr:`ansatz` property.
```

Comment:
I'm not sure whether we would want to refer to a private attribute in the release note here..
Maybe having the deprecation above is sufficient while also simply pointing out, that one can modify the excitation list of the provided ansatz?

@woodsp-ibm wdyt?

Reply:
The release notes are intended to be user facing and this I would agree that talking about some private attribute is out of place. Maybe something like this?

    Thus the excitation list cannot be set directly, rather if required this must be done
    directly on the anstaz via the :attr:`ansatz` property.